android/README.md

```diff
@@ -1,23 +1,43 @@
-## Building Keyple Android components
-
-If you want to build the keyple android components (aar plugins, apk example app), you need : 
+### Android components
+If you want to build the keyple android components (aar plugins), you need : 
 - Java JDK 1.8 or newer (OMAPI app requires java 1.8)
```

Comment:
about the _or newer_ I don't know: what I know is that it won't work with java 11